src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/AggregatedRssFeedProducer.java

```diff
@@ -49,7 +55,7 @@
     private static final String SELF_REL = "self";
     private static final String UTM_MEDIUM = "RSS";
     private static final String UTM_CAMPAIGN = "RSS";
-
+    private static final Set<String> EMPTY_AUTHORS_SET = ImmutableSet.of(StringUtils.EMPTY);
```

Comment:
The set has to be initialized with empty string to properly handle the case when [`getRss`](https://github.com/tdziurko/jvm-bloggers/pull/215/files#diff-bb1b427605a1cafe46d0c0b54e6c5125R79) will be called with `null` or empty `excludedAuthors` parameter. In that case JPA implementation will generate `WHERE` clause which will be inconsistent with business requirement. It will look like `... where blog.author not in (null)` which will produce empty `approvePosts`. To workaround the case a set with empty string should be provided to JPA to produce `WHERE` clause in the correct form: `... where blog.author not in ('')`